app/plugins/ConductrPlugin.scala

```diff
@@ -0,0 +1,20 @@
+package plugins
+
+import com.typesafe.conductr.bundlelib.play.StatusService
+import com.typesafe.conductr.bundlelib.play.ConnectionContext.Implicits
+import play.api.{ Application, Plugin, Logger }
+import play.api.libs.concurrent.Execution
+
+class ConductrPlugin extends Plugin {
+  
+  override def onStart() {
+    import Implicits.defaultContext
+    StatusService.signalStartedOrExit()
+    Logger.info("Application has started")
```

Comment:
I seem to remember us having a discussion about it, but I can't remember
the arguments of who held what point of view. Anyway I think you're
probably right that it makes sense to have a Logger injectable. If you make
a ticket (and if no one else objects) then I'll make it happen.